app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -99,6 +87,7 @@ class ChannelPostList extends PureComponent {
             },
             passProps: {
                 channelId,
+                intl,
```

Comment:
Can't the Thread screen use injectIntl itself? this.props.intl isn't used anywhere else in this component